recipes/disjoint-set/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "disjoint-set" %}
+{% set version = "0.7.1" %}
+{% set blake2_sha256 = "41a0caecf7d275fe5146c6afb80333c3e233388171432bf3bdf1e3633e990c0a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/{{ blake2_sha256[0:2] }}/{{ blake2_sha256[2:4] }}/{{ blake2_sha256[4:] }}/disjoint_set-{{ version }}.tar.gz
```

Comment:
Does the canonical url scheme not work:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14

I think we want to avoid having multiple hashes if possible. I'm not sure if the bots that update the recipe handle this well.

Reply:
It does! I overlooked replacing "-" with "_". I've corrected this.